whole-repo-tests/test_doctests.py

```diff
@@ -28,6 +28,7 @@ def test_doctests():
     env['PYTHONPATH'] = 'src'
 
     pip_tool(
-        'sphinx-build', '-W', '-b', 'doctest', '-d', 'docs/_build/doctrees',
+        'sphinx-build', '-n', '-W', '--keep-going', '-T',
```

Comment:
What was this change about?

Reply:
I had a couple of doctests fail, but CI would only show the first failure.  Now it shows all of them, and a full traceback if Sphinx has an internal error instead of logging to a file we can't retrieve.